src/event-notion/index.ts

```diff
@@ -452,9 +452,15 @@ export const pingForTAPandCSIDeadlines = async (notion: Client,
     if (event.properties.Name.type !== 'title') {
       return false;
     }
+    let tapStatus = 'TAP N/A';
+    let csiStatus = 'Intake Form N/A';
```

Comment:
Super tiny nitpick but I might move all calculations for CSI after everything for TAPs to separate them for readability. Also consider ternary expressions for boolean conditions if it's helpful for readability so we can use `const` variables.

```
const tapSelect = event.properties['TAP Status'].select;
const tapStatus = tapSelect ? tapSelect.name : 'TAP N/A';
```